src/middleware/authMiddleware.test.js

```diff
@@ -52,20 +52,4 @@ describe('authMiddleware', () => {
     login(mockRequest, mockResponse);
     expect(mockResponse.redirect).not.toHaveBeenCalledWith(process.env.TTA_SMART_HUB_URI);
   });
-
-  it('login should bypass HSES for cucumber tests', async () => {
-    const mockSession = jest.fn();
-    mockSession.userId = undefined;
-    const mockRequest = {
-      path: '/api/login',
-      session: mockSession,
-      headers: { cookie: `CUCUMBER_USER=${process.env.CUCUMBER_USER}` },
-    };
-    const mockResponse = {
-      redirect: jest.fn(),
-      sendStatus: jest.fn(),
-    };
-    login(mockRequest, mockResponse);
-    expect(mockResponse.redirect).toHaveBeenCalledWith(process.env.TTA_SMART_HUB_URI);
-  });
```

Comment:
My understanding is that we shouldn't need this test anymore because cucumber will no longer be using the `login` function.